pom.xml

```diff
@@ -714,7 +714,7 @@
       <dependency>
         <groupId>org.scalacheck</groupId>
         <artifactId>scalacheck_${scala.binary.version}</artifactId>
-        <version>1.12.5</version> <!-- 1.13.0 appears incompatible with scalatest 2.2.6 -->
+        <version>1.13.4</version>
```

Comment:
Yes, I remember this didn't work. Jenkins runs SBT; are you checking Maven locally? 
I'd like to update, sure, but want to make sure we don't trip over whatever I found before.

We could consider updating to scalatest 3.0.1 as well, in any event. As a test-only dependency it should be safe to more freely update it.